lib/Entity/Player.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MovingImage\Client\VMPro\Entity;
+
+use JMS\Serializer\Annotation\Type;
+
+class Player
+{
+    /** @Type("string") */
+    private $id;
+
+    /** @Type("string") */
+    private $name;
+
+    /** @Type("boolean") */
+    private $active = false;
+
+    public function getId(): ?string
+    {
+        return $this->id;
+    }
+
+    public function setId(int $id): self
```

Comment:
Property type of id and return type of getId() is string, so setter must also accept string and not int.

Reply:
thanks. done